src/ray/core_worker/core_worker.cc

```diff
@@ -2167,19 +2167,26 @@ void CoreWorker::HandleCancelTask(const rpc::CancelTaskRequest &request,
                                   rpc::SendReplyCallback send_reply_callback) {
   absl::MutexLock lock(&mutex_);
   TaskID task_id = TaskID::FromBinary(request.intended_task_id());
-  bool success = main_thread_task_id_ == task_id;
+  bool kill_success = main_thread_task_id_ == task_id;
```

Comment:
Could you rename this to `requested_task_running` (i.e. `requested_task_running= main_thread_task_id_ == task_id;`